mypy.ini

```diff
@@ -14,3 +14,7 @@ explicit_package_bases = True
 allow_redefinition = True
 strict_equality = True
 show_error_codes = True
+
+# Suppress the following messages:
+# By default the bodies of untyped functions are not checked, consider using --check-untyped-defs
+disable_error_code = annotation-unchecked
```

Comment:
(This is not strictly needed as those messages are informational and don't fail the check)